app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3 and command.split()[0] == "mv":
+        mv, file1_name, destination = command.split()
+        dirs = destination.split("/")[:-1]
+        file2_name = destination.split("/")[-1]
+
+        dir_path = ""
+        for folder in dirs:
+            dir_path = os.path.join(dir_path, folder)
+            if not os.path.exists(dir_path):
+                os.mkdir(dir_path)
```

Comment:
Ensure that you handle the case where the directory already exists to avoid errors, as mentioned in the checklist.